src/frontend/src/styles/main.css

```diff
@@ -949,6 +961,159 @@ a:hover,
   color: rgba(0, 0, 0, 0.5);
 }
 
+/**
+ * LandingPage component
+ */
+
+.c-landingPage {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--rs-main-content-width), 1fr);
+  grid-template-areas: "left right";
+
+  height: 100vh;
```

Comment:
should this not be min-height?

Reply:
And dvh instead of vh?

Reply:
I didn't know about `dvh` thanks!

I tried with `min-height`. But it creates a weird two scroll with the right page is higher than the screen. With `height` the scroll is smooth.

Reply:
But then something else is the issue. i you use height and the content happens to be higher as the actual layout, this might lead to issues

Reply:
The UI is ok, it's the scroll that seems weird.

This is with `min-height` (notice the pause in the scroll):

https://github.com/dfinity/internet-identity/assets/4550653/b70f5023-ab15-4f43-a1c8-6569f3e50537

This is with `height`:

https://github.com/dfinity/internet-identity/assets/4550653/8e284eee-0104-452f-bbd7-dfcdfb8a8c8a